File: SEM3/PYTHON/College_Codes/Control Statements/exercise3.py
flag=1

def gcd(a,b):
    if(a>b):
        num=a
        deno=b
    else:
        num=b
        deno=a
    
    remainder = num%deno

    while(remainder!=0):
        num=deno
        deno=remainder
        remainder = num%deno

    gcd=deno
    
    return gcd

while(flag==1):
    a=int(input("Enter your 1st number: "))
    b=int(input("Enter your 2nd number: "))
    
    print("Result: "+ str(gcd(a,b)))

    intake=input("Do you wish to continue? (Press y to continue or any key to exit.): ")

    if(intake=="Y" or intake=="y"):
        flag=1
    else:
        flag=0
        break

File: SEM2/College_Codes/Functions/Exercise3.c
#include<stdio.h>
int gcd(int a,int b)
{
    int num,deno,remainder,gcd;
    if(a>b)
    {
        num=a; deno=b;
    }
    else
    {
        num=b; deno=a;
    }
    remainder = num%deno;
    while(remainder!=0)
    {
        num=deno;
        deno=remainder;
        remainder = num%deno;
    }
    gcd=deno;
    return gcd;
}
int lcm(int a,int b)
{
    int lcm;
    lcm = a*b / gcd(a,b);
    printf("GCD of %d & %d is %d\n",a,b,gcd(a,b));
    printf("LCM of %d & %d is %d\n",a,b,lcm);
}
int main()
{
    int a,b;
    printf("Enter any number: ");
    scanf("%d%d",&a,&b);
    lcm(a,b);
    return 0;
}